rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1055,20 +1052,20 @@ public long getLockWhitelistEntryByAddress(Object[] args) {
         try {
             addressBase58 = (String) args[0];
         } catch (Exception e) {
-            logger.warn("Exception in getLockWhitelistEntryByAddress", e);
-            return LOCK_WHITELIST_INVALID_ADDRESS_FORMAT_ERROR_CODE;
+            logger.warn("[getLockWhitelistEntryByAddress] Exception in getLockWhitelistEntryByAddress", e);
```

Comment:
Awesome! Looks much better with the pattern: logger.type("[method]"...). 
WDYT if we add a variable with some information and remove "Exception in getLockWhitelistEntryByAddress" (it seems redundant with the name of the method):

`logger.warn("[removeLockWhitelistAddress] {}", e.getMessage(), e);`

Reply:
Done, thanks!